Mathlib/CategoryTheory/Localization/Predicate.lean

```diff
@@ -168,9 +168,20 @@ lemma isoOfHom_id_inv (X : C) (hX : W (𝟙 X)) :
   rw [← cancel_mono (isoOfHom L W (𝟙 X) hX).hom, Iso.inv_hom_id, id_comp,
     isoOfHom_hom, Functor.map_id]
 
+variable {W}
+
+lemma Construction.wIso_eq_isoOfHom {X Y : C} (f : X ⟶ Y) (hf : W f) :
+    Construction.wIso f hf = isoOfHom W.Q W f hf := by ext; rfl
+
+lemma Construction.winv_eq_isoOfHom_inv {X Y : C} (f : X ⟶ Y) (hf : W f) :
+    Construction.winv f hf = (isoOfHom W.Q W f hf).inv :=
```

Comment:
Should this be called `wInv` instead of `winv`? (out of scope of this PR ofc)

Reply:
Thanks! As this is mostly unused (this is part of the very internals of the localization API which should not be used too much), I have done the renaming.